docs/src/man/comparisons.md

```diff
@@ -235,6 +235,66 @@ The table below compares more advanced commands:
+| Reduce multiple values      | `df[, list(mean(x)), by = list(id) ]`             | `combine(groupby(df, :id), :x => mean)`   |
+| Add new columns (in place)  | `df[, x_mean := mean(x), by = list(id) ]`         | `transform!(groupby(df, :id), :x => mean)`|
+| Pick & transform columns    | `df[, list(x_mean = mean(x), y), by = list(id) ]` | `select(groupby(df, :id), :x => mean, :y)`|
+
+### More advanced commands
+
+| Operation                         | data.table                                                                                 | DataFrames.jl                                                               |
+|:----------------------------------|:-------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------|
+| Complex Function                  | `df[, list(mean(x, na.rm = T)) ]`                                                          | `combine(df, :x => x -> mean(skipmissing(x)))`                              |
+| Transform certain rows (in place) | `df[x<=0, x:=0 ]`                                                                          | `df.x[df.x .<= 0] .= 0`                                                     |
+| Transform several columns         | `df[, list(max(x), min(y)) ]`                                                              | `combine(df, :x => maximum, :y => minimum)`                                 |
+|                                   | `df[, lapply(.SD, mean), .SDcols = c("x", "y") ]`                                          | `combine(df, [:x, :y] .=> mean)`                                            |
+|                                   | `df[, lapply(.SD, mean), .SDcols = patterns("x*") ]`                                       | `combine(df, names(df, r"^x") .=> mean)`                                    |
+|                                   | `df[, unlist(lapply(.SD, function(x) c(max=max(x), min=min(x)))), .SDcols = c("x", "y") ]` | `combine(df, ([:x, :y] .=> [maximum minimum])...)`                          |
+| Multivariate function             | `df[, list(cor(x,y)) ]`                                                                    | `transform(df, [:x, :y] => cor)`                                            |
+| Row-wise                          | `df[, min_xy := min(x, y), by = 1:nrow(df)]`                                               | `transform(df, [:x, :y] => ByRow(min))`                                     |
+|                                   | `df[, argmax_xy := which.max(.SD) , .SDcols = patterns("x*"), by = 1:nrow(df) ]`           | `transform(df, AsTable(r"^x") => ByRow(argmax))`                            |
+| DataFrame as output               | `df[, .SD[, list(value = min(x), max(x))] ]`                                               | `combine(df, :x => (x -> (value = [minimum(x), maximum(x)],)) => AsTable)`  |
```

Comment:
these operations do not produce the same result. Also I think that the DataFrames.jl part is overly complex, you could have just written `combine(df, :x => (x -> [minimum(x), maximum(x)]) => :value)` to get the same.

Reply:
I think you told me earlier to write `:x => (x -> (value = [minimum(x), maximum(x)],))` 
The goal of this line is to emulate how data.table can process sub-tables inside of a data.table. I thought this was similar to the written code in dataframes. 
I think writing `combine(df, :x => (x -> [minimum(x), maximum(x)]) => :value)` is a little different in spirit. 

I changed the code so it does give the right answer though.

Reply:
OK - we can leave it as is (the code is clear for me - I was unsure if it will be easy to read for people who learn DataFrames.jl)